kinto/__main__.py

```diff
@@ -52,13 +54,14 @@ def main(args=None):
                               default=False)
     parser_start.set_defaults(which='start')
 
-    args = vars(parser.parse_args())
+    args = vars(parser.parse_args(args))
```

Comment:
Note: /me wonders how it could even work before =)

Nit: Maybe we could create another var instead of overriding the existing `args` one? eg. `parsed_args`